NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -70,16 +77,21 @@ let SkipTimeInterval: Double = 15
     override public func viewDidLoad() {
         super.viewDidLoad()
 
+        self.view.backgroundColor = NYPLColor.primaryBackgroundColor
+        self.activityIndicator.color = NYPLColor.disabledFieldTextColor
         self.audiobookManager.audiobook.player.registerDelegate(self)
         self.audiobookManager.networkService.registerDelegate(self)
         self.audiobookManager.networkService.fetch()
 
-        self.gradient.frame = self.view.bounds
-        let startColor = UIColor(red: (210 / 255), green: (217 / 255), blue: (221 / 255), alpha: 1).cgColor
-        self.gradient.colors = [ startColor, UIColor.white.cgColor]
-        self.gradient.startPoint = CGPoint.zero
-        self.gradient.endPoint = CGPoint(x: 1, y: 1)
-        self.view.layer.insertSublayer(self.gradient, at: 0)
+        if #available(iOS 12.0, *),
+           UIScreen.main.traitCollection.userInterfaceStyle == .light {
+            self.gradient.frame = self.view.bounds
+            let startColor = UIColor(red: (210 / 255), green: (217 / 255), blue: (221 / 255), alpha: 1).cgColor
+            self.gradient.colors = [ startColor, UIColor.white.cgColor]
+            self.gradient.startPoint = CGPoint.zero
+            self.gradient.endPoint = CGPoint(x: 1, y: 1)
+            self.view.layer.insertSublayer(self.gradient, at: 0)
+        }
```

Comment:
what happens in the `else` case? even if we don't need anything there, can you add a comment about why we are ok? from reading the code it looks like we're missing handling the "non-ios-12 / dark" case

Reply:
Nice catch, thanks